src/server/views/app.handlebars

```diff
@@ -171,15 +174,14 @@
         .catch(console.error);
     }
 
-    const urlParams = new URLSearchParams(window.location.search);
     const asyncApiUrl = urlParams.get('load') || urlParams.get('url');
-
     if (asyncApiUrl) {
       document.getElementById('fetch-document-url').value = asyncApiUrl;
       loadAsyncApi(asyncApiUrl);
     }
 
     App.addEventListener('templateChange', (e) => {
+      console.log('dupa')
```

Comment:
You might want to remove the console log before it gets merged

Reply:
Haha 😂  Thanks man!